plugins/inventory/xen_orchestra.py

```diff
@@ -72,6 +80,8 @@
     kube_nodes: "'kube_node' in tags"
 compose:
     ansible_port: 2222
+use_vm_uuids: False
+use_host_uuids: True
```

Comment:
YAML booleans should always be lowercase.
```suggestion
use_vm_uuids: false
use_host_uuids: true
```